invoice.go

```diff
@@ -47,11 +60,13 @@ type InvoiceUpcomingInvoiceItemParams struct {
 // For more details see https://stripe.com/docs/api#create_invoice, https://stripe.com/docs/api#update_invoice.
 type InvoiceParams struct {
 	Params              `form:"*"`
+	AutoAdvance         *bool    `form:"auto_advance"`
```

Comment:
I think `auto_advance` is just a "finalize" parameter.

Reply:
Where did you see that one? `auto_advance` is the new world and has been planned for months. Just confirmed with the team internally that it's `auto_advance` and `finalize` does not exist as a parameter (but is a method)

Reply:
(Spoke with Remi offline: this seems to be a documentation problem in that `auto_advance` isn't showing up in some places at the moment. Fine by me if we know it's supposed to be here.)